### PR TITLE
[E2ETest][UWP] Adding completion file logic to UWP E2ETest application

### PR DESCRIPTION
**Pull request by gb92**
Added logic to create a completion file at the end of the automated test run

### CODE REVIEW

File: e2etest/UWP.E2ETest/UI/TestPage.xaml.cs [hunk 1]
```diff
@@ -67,6 +69,17 @@ public TestPage()
 
         public async void EndRun(TestHarness harness)
         {
+            string completionFileName = "uwp_e2etest_completion.txt";
+            StorageFolder storageFolder = ApplicationData.Current.LocalFolder;
+            string filePath = storageFolder.Path + completionFileName;
```
Comment (fabiocav) [inline e2etest/UWP.E2ETest/UI/TestPage.xaml.cs]: Use `Path.Combine` instead.

File: e2etest/UWP.E2ETest/UI/TestPage.xaml.cs [hunk 2]
```diff
@@ -2,15 +2,17 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // ----------------------------------------------------------------------------
 
+using Microsoft.WindowsAzure.MobileServices.TestFramework;
```
Comment (dhei) [inline e2etest/UWP.E2ETest/UI/TestPage.xaml.cs]: nitpick: System using statements should be prior to other using statements.

**Comment by dhei** [comment]
Looks good :+1:

**Comment by fabiocav** [comment]
I had a minor comment here as well. Would be a quick one to address.

**Pull request merged**